app/main.py

```diff
@@ -1,3 +1,18 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    @classmethod
+    def from_dict(cls, course_dict):
+        return OnlineCourse(name=course_dict.get("name"),
```

Comment:
It's better to use `cls` in classmethods

Reply:
fixed